spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/builder/ExportGenerator.java

```diff
@@ -177,4 +250,17 @@ private PoolDTO createPool(String ownerKey, ProductDTO product) throws ApiExcept
 
         return ownerApi.createPool(ownerKey, pool);
     }
+
+    public ContentDTO getContent() {
```

Comment:
This won't work correctly for exports that attach more than one product or content.